src/test/java/com/example/homework10/integrationtest/MusicApiIntegrationTest.java

```diff
@@ -0,0 +1,83 @@
+package com.example.homework10.integrationtest;
+
+import com.github.database.rider.core.api.dataset.DataSet;
+import com.github.database.rider.spring.api.DBRider;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.nio.charset.StandardCharsets;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@DBRider
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class MusicApiIntegrationTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Test
+    @Transactional
+    @DataSet("datasets/music.yml")
+    void ミュージックが全て取得できること() throws Exception {
+        String response = mockMvc.perform(MockMvcRequestBuilders.get("/music"))
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andReturn().getResponse().getContentAsString(StandardCharsets.UTF_8);
+
+        JSONAssert.assertEquals("""
+                [
+                    {
+                        "id":1,
+                        "title":"The Beginning",
+                        "singer":"OneOkRock"
```

Comment:
全部大文字で半角スペースありが正式名称かなと思います。
```suggestion
                        "singer":"ONE OK ROCK"
```

Reply:
[024ec63](https://github.com/yuu3415/homework10/pull/4/commits/024ec637740415ccfb324fcb93b8c9b1fe76d4c1)
有難うございます。
修正させて頂きました。